src/loaders.py

```diff
@@ -9,52 +11,80 @@
 from steppy.base import BaseTransformer
 
 from .retinanet import DataEncoder
-from .pipeline_config import MEAN, STD
+from .pipeline_config import MEAN, STD, SEED
+from .utils import get_target_size
 
 
-class RandomSubsetSampler(Sampler):
-    def __init__(self, data_size, sample_size):
-        self.data_size = data_size
-        self.sample_size = sample_size
+class FixedSizeSampler(Sampler):
+    def __init__(self, images_data, *,  sample_size=None, **kwargs):
+        self.images_data = images_data.sample(frac=1, random_state=SEED)
```

Comment:
@taraspiotr I think there should be some explicit shuffle parameter for this.

Reply:
@jakubczakon ok